ush/gefs_sndp.sh

```diff
@@ -61,7 +61,7 @@ EOF
 
     if [ $SENDCOM = 'YES' ]
     then 
-      if [ $SENDDBN = 'YES' ] ; then
+      if [ $SENDDBN_NTC = 'YES' ] ; then
```

Comment:
Why the change? It seems counter to the request to change from SENDDBN_GB2. Is it confirmed that NCO approves of this change?

Reply:
Add Dingchen to the reviewer, This is what he changed.

Reply:
Rereading the Implementation Standards, this seems correct. But I am going to double-check with Steven in the morning.